### PR TITLE
[ATOM-16524] MCPP should not add new line

### PR DESCRIPTION
**Pull request by galibzon**
characters at the end of comment lines

This version will be "mcpp-2.7.2_az.3-rev1-<platform>"

Previously comment lines like this:
---------------------------
// Line 1
// Line 2
// Line 3
---------------------------

Would appear, after preprocessing, as
---------------------------
// Line 1

// Line 2

// Line 3

---------------------------

With this fix the number of lines is preserved per the original.

Signed-off-by: garrieta <garrieta@amazon.com>

### CODE REVIEW

File: package-system/mcpp/mcpp_2.7.2-az.3.patch [hunk 1]
```diff
@@ -0,0 +1,897 @@
+diff --git a/config/config.guess b/config/config.guess
+index 396482d..e3ef63f 100644
+--- a/config/config.guess
++++ b/config/config.guess
```
Comment (lawsonamzn) [inline package-system/mcpp/mcpp_2.7.2-az.3.patch:4]: Is there any way to not modify config.guess?  Its license is GPL, and modifying it causes licensing concerns.
Is this maybe just an update of it?

File: package-system/mcpp/.gitattributes [hunk 2]
```diff
@@ -1 +1,2 @@
-mcpp-2.7.2-az.2.patch text eol=lf
\ No newline at end of file
+mcpp-2.7.2-az.2.patch text eol=lf
+mcpp_2.7.2-az.3.patch text eol=lf
```
Comment (lawsonamzn) [inline package-system/mcpp/.gitattributes:2]: please add a newline at the end of this file.

File: package-system/mcpp/mcpp_2.7.2-az.3.patch (hunk 1 above)
Comment (lawsonamzn) [reply package-system/mcpp/mcpp_2.7.2-az.3.patch:4]: did we maybe pull in a new version of mcpp or something and get a bunch of junk added?

Comment (galibzon) [reply package-system/mcpp/mcpp_2.7.2-az.3.patch:4]: This patch contains the same content of the original patch, + my new code in 'support.c'

File: package-system/mcpp/.gitattributes (hunk 2 above)
Comment (galibzon) [reply package-system/mcpp/.gitattributes:2]: done.

File: package-system/mcpp/mcpp_2.7.2-az.3.patch (hunk 1 above)
Comment (jeremyong-az) [reply package-system/mcpp/mcpp_2.7.2-az.3.patch:4]: Why do changes to `config.guess` show up in the diff then, given that your changes only purportedly change `support.c`?

**Comment by galibzon** [comment]
Closing the pull request. The MCPP folders need deep cleaning before introducing my changes to "support.c".

**Pull request closed without merging**